pontoon/base/static/js/translate.js

```diff
@@ -1084,11 +1091,27 @@ var Pontoon = (function (my) {
 
       var status = this.getEntityStatus(entity),
           translation = entity.translation[0].string;
+
       entity.ui.addClass(status);
       entity.ui.find('.translation-string')
         .html(this.doNotRender(translation || ''));
 
       this.updateProgress();
+
+      // Update has-suggestions status
```

Comment:
This is epic, but also the only reliable way to get this information. We should probably bring all translations to the client with get-entities and then we don't need to make requests just to get this information.

It's a good enough solution UX wise, because we don't need this information instantly anyways.